resource-managers/kubernetes/core/src/main/scala/org/apache/spark/scheduler/cluster/k8s/KubernetesClusterSchedulerBackend.scala

```diff
@@ -217,7 +217,7 @@ private[spark] class KubernetesClusterSchedulerBackend(
         .watch(new ExecutorPodsWatcher()))
 
     allocatorExecutor.scheduleWithFixedDelay(
-      allocatorRunnable, 0L, podAllocationInterval, TimeUnit.SECONDS)
+      allocatorRunnable, 0L, podAllocationInterval.toLong, TimeUnit.MILLISECONDS)
```

Comment:
Just checking this definitely returns ms? Looks good then.

Reply:
Why not use `conf.getTimeAsMs` for `podAllocationInterval` ?

Reply:
Done. That's much better, thanks!